src/shared/port-rpc.js

```diff
@@ -123,15 +135,19 @@ export class PortRPC {
   /**
    * Send an RPC request via the connected port.
    *
+   * If this client is not yet connected to a port, the call will be queued and
+   * sent when {@link connect} is called.
+   *
    * If the final argument in `args` is a function, it is treated as a callback
    * which is invoked with the response in the form of (error, result) arguments.
    *
    * @param {CallMethod} method
    * @param {any[]} args
    */
   call(method, ...args) {
-    // TODO - What should happen here if the port is not connected? Buffer
-    // method calls until the port is connected?
+    if (!this._port) {
+      this._pendingCalls.push([method, args]);
```

Comment:
Personally, I would check if the port is `null` only once, and leave just `this._destroyed` in the next `if` statement.
```suggestion
      this._pendingCalls.push([method, args]);
      return;
```

Reply:
Drat, I missed this before merging. I'll have to come back to this.